notebooks/en/_toctree.yml

```diff
@@ -28,3 +28,6 @@
     title: Prompt tuning with PEFT
   - local: semantic_cache_chroma_vector_database
     title: Implementing semantic cache to improve a RAG system.
+  - local: llm_judge
+    title: LLM-as-a-judge 🧑‍⚖️ - Building an automated and versatile evaluation system 
```

Comment:
Let's not use emojis in the titles in yaml, it can break the build. Also, if possible, let's make this title a bit shorter.

Reply:
Thanks! Also I was wondering, does ':' in the title break the build as well ? (given that there is already a ':' after 'title')